netbox/ipam/tests/test_ordering.py

```diff
@@ -0,0 +1,290 @@
+from django.test import TestCase
+
+from ipam.choices import IPAddressStatusChoices, PrefixStatusChoices
+from ipam.models import IPAddress, Prefix, VRF
+
+import netaddr
+
+
+class PrefixOrderingTestCase(TestCase):
+    vrfs = None
+
+    def setUp(self):
+        vrfa = VRF(name="VRF A")
```

Comment:
Should be able to simplify this as:

```python
self.vrfs = (
    VRF(name="VRF A"),
    VRF(name="VRF B"),
    VRF(name="VRF C"),
)
VRF.objects.bulk_create(self.vrfs)
```

Reply:
Yeah, it should.  I will get on it later today